### ISSUE TITLE
[Bug Report][2.6.7] Expansion Panel uses `aria-expanded` attribute on a div

### ISSUE THREAD

**Issue by marjev** [vuetifyjs/vuetify#15514]
### Environment
**Vuetify Version:** 2.6.7
**Vue Version:** 2.7.7
**Browsers:** Chrome 103.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
1. Install a tool similar to [Axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US). Alternatively, add a Vuetify expansion panel into a [Storybook](https://storybook.js.org/) story and install the [Accessibility addon](https://storybook.js.org/addons/@storybook/addon-a11y/).
2. Screen a page that includes an Expansion panel.
 

### Expected Behavior
a11y screening shouldn't fail on the following [criteria](https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=axeAPI).
The button should be the element that has `aria-expanded` attribute.

### Actual Behavior
Div element wrapping an expansion panel (div with the `v-expansion-panel` class on it) has `aria-expanded` attribute attached to it.

### Reproduction Link
<a href="https://codepen.io/marjev/pen/MWVvRmm" target="_blank">https://codepen.io/marjev/pen/MWVvRmm</a>

### Other comments
See [this Vanilla JS example](https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion.html) which demonstrates HTML which doesn't fail this screening.

**Comment by Harm-Nullix** [comment]
This is also the case in the NEXT.vuetify:
![image](https://user-images.githubusercontent.com/80104552/219024423-1044cd1d-c6bf-4a41-b827-30ce9a1695ec.png)

```
    "axe-core": "4.6.3"
    "vuetify": "3.1.4"
```
